lesson_15/sources/team.cpp

```diff
@@ -0,0 +1,40 @@
+#include "../headers/team.hpp"
+
+team::team(std::string_view name) : name_{ name }
+{
+}
+
+std::string_view team::get_name() const
+{
+    return name_;
+}
+
+void team::add_player(player* player)
+{
+    if (num_players_ < team_max_players_num)
+    {
+        players_[num_players_++] = player;
+        player->team_ = this;
```

Comment:
it's better to create setter

Reply:
ok.
in this case anyone can set "team" to a "player" but "team" will not know about this change.
and also I done in for using "friend" relationship between classes